dashboard/requirements.txt

```diff
@@ -0,0 +1,11 @@
+requests
+pylint
```

Comment:
pylint definitely not needed, because we'd end up adding it to the image which would be uneccessary.

If you want to use pylint i think it's best to use it locally!